bors.toml

```diff
@@ -8,4 +8,5 @@ status = [
   "test_daemons_release_mode (ubuntu-latest)",
   "test_daemons_release_mode (macos-latest)",
   "daemons_arm_build",
+  "block_fixup_commit_merge",
```

Comment:
I think it is the job name that is being tested against, not the step name.

Reply:
fixing up, will verify whether bors blocks it :D

Reply:
It won't because you've declared it only for `pull_request` events but bors pushes on a `staging` branch and then only checks that are triggered by `push` events are executed.

We can either:

- See if the action works with `push` events
- Put a GitHub merge restriction in place which will have bors fail because it can't trigger the merge through the API

Reply:
the second option seems nicer - I'll have a quick look at it!

Reply:
I'll activate the check.